samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -263,6 +265,14 @@ public long getChangelogMinCompactionLagMs(String storeName) {
     return getLong(minCompactLagConfigName, getDefaultChangelogMinCompactionLagMs());
   }
 
+  /**
+   * Helper method to get the default RocksDB max manifest file size in bytes, which is default to 1GB.
+   */
+  public long getDefaultMaxManifestFileSizeBytes() {
+    String defaultMaxManifestFileSizeConfigName = String.format(ROCKSDB_MAX_MANIFEST_FILE_SIZE, "default");
```

Comment:
Instead of using string "default", use a constant. In the docs, describe the purpose of using `default`. It might not be clear to the reader that `stores.default` is a samza config which applies to all stores.

Please add details the the method docs regarding default store config.

Reply:
Thanks for the suggestion. I updated the method doc and the configuration md file.